lib/commands/context.js

```diff
@@ -279,7 +289,12 @@ extensions.onPageChange = async function (pageArray) {
         })[0];
       };
 
-      return !_.isEqual(item(this.contexts), item(pageArray));
+      // need to map the page ids to context ids
+      let contextArray = _.map(pageArray, (arr) => {
```

Comment:
same one-liner trick here if you want to consolidate comments. otherwise this is fine.